source/CardGames.py

```diff
@@ -250,6 +250,14 @@ def rewardPot(self, player):
     player.addMoney(self.money)
     self.money = 0
 
```

Comment:
This looks good. Leaves plenty of room for expansion if we have time possibly next sprint.